### PR TITLE
Fixed Broken Link in Get-AzDtlAllowedVMSizesPolicy.md

### PR DESCRIPTION
**Pull request by isjwuk**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by msJinLei** [comment]
@tanmayeekamath could you help to review the document change? Thanks.

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/DevTestLabs/DevTestLabs/help/Get-AzDtlAllowedVMSizesPolicy.md [hunk 1]
```diff
@@ -88,6 +88,6 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [Set-AzDtlAllowedVMSizesPolicy](./Set-AzDtlAllowedVMSizesPolicy.md)
 
-[Azure Development Test Lab Cmdlets](./Az.DevTestLabs.md)
```
Comment (msJinLei) [inline src/DevTestLabs/DevTestLabs/help/Get-AzDtlAllowedVMSizesPolicy.md]: why this link is broken?

Comment (isjwuk) [reply src/DevTestLabs/DevTestLabs/help/Get-AzDtlAllowedVMSizesPolicy.md]: When you click on the existing link (see https://docs.microsoft.com/en-us/powershell/module/az.devtestlabs/get-azdtlallowedvmsizespolicy?view=azps-3.1.0 ) it returns a 404.

Comment (msJinLei) [reply src/DevTestLabs/DevTestLabs/help/Get-AzDtlAllowedVMSizesPolicy.md]: > When you click on the existing link (see https://docs.microsoft.com/en-us/powershell/module/az.devtestlabs/get-azdtlallowedvmsizespolicy?view=azps-3.1.0 ) it returns a 404.

Thanks for discovering the issue. It is caused by Microsoft doc fails to generate the link to upper layer document (catalog) file correctly. 
As the user can access catalog by side column, it is better to just remove the link in Get-AzDtlAllowedVMSizesPolicy.md.

**Pull request merged**